scmtfvc.properties

```diff
@@ -37,3 +36,9 @@ defaults.mavenArtifactId=sonar-scm-tfvc-plugin
 2.1.2.downloadUrl=https://github.com/SonarQubeCommunity/sonar-scm-tfvc/releases/download/2.1.2/sonar-scm-tfvc-plugin-2.1.2.jar
 2.1.2.date=2015-05-02
 2.1.2.changelogUrl=https://github.com/SonarQubeCommunity/sonar-scm-tfvc/issues?q=milestone%3A2.1.2
+
+2.2.description=Fix compatibility for SQ 7.3 and above
+2.2.sqVersions=[7.0,7.9.*]
```

Comment:
I have to admit that I don't know if it's really compatible with 7.0. The build definition is currently using 6.7. This would mean that it should even work with 6.7. But I can't try it.
Should I better specify the known lower limit (7.3)?

Reply:
For the bottom of the range we have 2 options (okay 3 - we could test ;-p) : be conservative and use 7.3, or be optimistic and use 7.0 and see what users report. Since there _is_ a version that's compatible with 7.2 and below I'd vote for conservative here, but let's say I have a 49% stake in this. Note that if you do start at 7.3 you'll want to move 2.1.2 back into the `publicVersions` list.

For the top of the range, I _will_ urge you to be optimistic and use `LATEST`. It's _possible_ that breaking API changes will be made in 8.0 (that's _always_ possible) but they're not usually made out of the blue. Anyway, if 8.0 does break compatibility, we can catch that up at the time but I'd let it be declared compatible by default.

Reply:
done